core/src/main/java/com/linecorp/armeria/server/PathMapping.java

```diff
@@ -25,6 +25,12 @@
  */
 interface PathMapping {
 
+    /**
+     * Returns a newly-created {@link PathMapping} which adds the specified {@code prefix}
```

Comment:
```suggestion
     * Returns a newly-created {@link PathMapping} which prepends the specified {@code prefix}
```

Reply:
Decided to go with 'add prefix' because 'prefix' implies something that comes before already.